lib/Mojo/WebSocketProxy/Backend/JobAsync.pm

```diff
@@ -58,29 +58,11 @@ sub new {
     my ($class, %args) = @_;
     # Avoid holding these - we only want the Job::Async::Client instance, and everything else
     # should be attached to the loop (which sticks around longer than we expect to).
-    my $loop = delete $args{loop};
+    my $loop   = delete $args{loop};
     my $jobman = delete $args{jobman};
```

Comment:
@nael-binary  These variables are not used anymore and just deleting from args.

Reply:
this is needed here, not the variables themselves, but the deleting part for `$args` so maybe we can just delete without the need of assigning them to any variable